### PR TITLE
Fix gradle rook - RK-208

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**

### CODE REVIEW

File: java-gradle/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This sample may be out of date. If you face any issues, please reach out to mail
      ```
 3. Run using gradle:
     ```bash
-    $ ./gradlew run
+    $ gradle run
```
Comment (LiranLast) [inline java-gradle/README.md]: @ManuelNemirovsky I think @MickaelAlliel wanted it the other way around

Comment (ManuelNemirovsky) [reply java-gradle/README.md]: Done

Comment (ManuelNemirovsky) [reply java-gradle/README.md]: Working now.
`./gradlew fatJar run` needs to be run

File: java-gradle/build.gradle [hunk 2]
```diff
@@ -33,9 +32,11 @@ task fatJar(type: Jar) {
     with jar
 }
 
-apply plugin: 'application'
-applicationDefaultJvmArgs = ["-javaagent:rook.jar"]
+ apply plugin: 'application'
+ applicationDefaultJvmArgs = ["-javaagent:${configurations.rookoutAgent.singleFile}"]
 
 run {
+    println 'test'
```
Comment (itielshwartz) [inline java-gradle/build.gradle]: remove

Comment (ManuelNemirovsky) [reply java-gradle/build.gradle]: Done

**Pull request merged**